runtimevar/httpvar/httpvar.go

```diff
@@ -193,14 +193,18 @@ func equivalentError(err1, err2 error) bool {
 // service.
 type watcher struct {
 	client   *http.Client
-	endpoint *url.URL
+	endpoint string
```

Comment:
Can we leave this as a `*url.URL` please?

Reply:
Sure, didn't see the point of calling `.String()` on every GET.